src/DynamoCore/Properties/Resources.en-US.resx

```diff
@@ -747,4 +747,32 @@ Parameter name: {0}</value>
   <data name="DuplicatedNewerPackage" xml:space="preserve">
     <value>A newer version of the package called {0} version {2} was found at {1} with version {3}. The newer version has been ignored.</value>
   </data>
+  <data name="PackageStatePendingUnload" xml:space="preserve">
+    <value>Scheduled to be unloaded</value>
+  </data>
+    <data name="PackageStateError" xml:space="preserve">
+    <value>Error</value>
+  </data>
+    <data name="PackageStateLoaded" xml:space="preserve">
+    <value>Loaded</value>
+  </data>
+  <data name="PackageStateUnloaded" xml:space="preserve">
+    <value>Unloaded</value>
+  </data>
+  <data name="PackageStatePendingUnloadTooltip" xml:space="preserve">
+    <value>Scheduled to be unloaded.
+This package will be unloaded after the next Dynamo restart.</value>
+  </data>
+    <data name="PackageStateErrorTooltip" xml:space="preserve">
+    <value>Unloaded.
+This package has not been loaded due to an unexpected error.</value>
+  </data>
+    <data name="PackageStateLoadedTooltip" xml:space="preserve">
+    <value>Loaded.
+This package is loaded and ready to be used.</value>
+  </data>
+    <data name="PackageStateUnloadedTooltip" xml:space="preserve">
+    <value>Unloaded.
+This package has not been loaded because another conflicting package was loaded before it.</value>
```

Comment:
Same as above, we may be able to specify which package caused the current one to get into an Unloaded state